week6/Products.cs

```diff
@@ -0,0 +1,48 @@
+using Dapper;
+using MySql.Data.MySqlClient;
+
+namespace HackYourFuture.Week6;
+
+
+public interface IProductRepository
```

Comment:
you can further split the `IProductRepository` in a `IProductRepository.cs` file. Then you will have a dedicated file for the interface and a dedicated file for the implementation which will be `ProductRepository.cs`

Reply:
This would be extra, I noticed it was not part of the homework 🙏